tests/conftest.py

```diff
@@ -200,9 +200,9 @@ def repos_tmpdir(tmpdir, _session_repos):
         path: zephyr
 
     '''
-    kconfiglib, tagged_repo, net_tools, zephyr = [
+    kconfiglib, tagged_repo, net_tools, zephyr = (
         os.path.join(_session_repos, x) for x in
-        ['Kconfiglib', 'tagged_repo', 'net-tools', 'zephyr']]
+        ['Kconfiglib', 'tagged_repo', 'net-tools', 'zephyr'])
```

Comment:
I'm curious what was the warning here...

Reply:
It was actually a [deprecated rule](https://docs.astral.sh/ruff/rules/unpacked-list-comprehension/) itself, which was still "automagically" fixed with `ruff check --fix`.

I've ignored this rule.